prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -79,6 +79,14 @@ ssize_t rxr_pkt_post_data(struct rxr_ep *rxr_ep,
 
 	data_pkt->hdr.rx_id = tx_entry->rx_id;
 
+	/*
+	 * If delivery complete protocols were used, we need to
+	 * set the data hdr flags to a non-zero value to handle
+	 * the logic in rxr_pkt_handle_data_send_completion.
+	 */
+	if (tx_entry->rxr_flags & RXR_DELIVERY_COMPLETE_REQUESTED)
+		data_pkt->hdr.flags = 1;
```

Comment:
At the very least, we need a flag to indicate DATA packet is DC, not just use 1.

Reply:
But adding the flag result in wire protocol change. Is it okay for me to do that?

Reply:
This also change the packet content, thus is also a wire protocol change.

Reply:
See comment below, the flag in the tx_entry that can be used here so I don't understand why this needs to be added to the header.

Reply:
Wire protocol changes is avoided now. Resolving.